### PR TITLE
Fix: ErrorPanel "X" icon

### PR DESCRIPTION
**Pull request by ededals**
Pull request for ErrorPanel "X" [#355](https://github.com/taskcluster/taskcluster-web/issues/355)

### CODE REVIEW

File: src/components/ErrorPanel/index.jsx [hunk 1]
```diff
@@ -23,6 +23,7 @@ import palette from '../../utils/palette';
       padding: `0 ${2 * theme.spacing.unit}px`,
       display: 'flex',
       justifyContent: 'space-between',
+      alignItems: 'center',
```
Comment (helfi92) [inline src/components/ErrorPanel/index.jsx]: This looks like it's the only change necessary to make it work :)

File: src/components/ErrorPanel/index.jsx [hunk 2]
```diff
@@ -61,6 +62,10 @@ import palette from '../../utils/palette';
       fontSize: theme.typography.pxToRem(15),
       fontWeight: theme.typography.fontWeightRegular,
     },
+    iconbutton:{
+      height: 50,
+      width: 50 
+    },
```
Comment (helfi92) [inline src/components/ErrorPanel/index.jsx]: Remove `iconButton`.

File: src/components/ErrorPanel/index.jsx [hunk 3]
```diff
@@ -134,7 +139,9 @@ export default class ErrorPanel extends Component {
           )}>
           {markdown}
           {onClose && (
-            <IconButton onClick={onClose}>
+            <IconButton 
+              className={classNames(classes.iconbutton)}
```
Comment (helfi92) [inline src/components/ErrorPanel/index.jsx]: Remove `className`.

**Comment by ededals** [comment]
I made the requested changes to the file.
Please see if it is ok now.

**Pull request merged**